### PR TITLE
Add docker support

### PR DESCRIPTION
**Pull request by noahnsimbe**
**Description**
Adding docker support and updating the README file with instructions on how to run using docker.

**Issues addressed**

- [ ] Resolves https://github.com/MetridashCore/Batman/issues/2

### CODE REVIEW

**Review by kanakdesai — COMMENTED** [review]
Port 3000 is used for both the backend as well as the font end? I might be wrong, but using same port for backend as well as the frontend might cause problems.

**Comment by iiio2** [comment]
> Port 3000 is used for both the backend as well as the font end? I might be wrong, but using same port for backend as well as the frontend might cause problems.

@NoahNsimbe pls respond

**Comment by noahnsimbe** [comment]
> > Port 3000 is used for both the backend as well as the font end? I might be wrong, but using same port for backend as well as the frontend might cause problems.
> 
> @NoahNsimbe pls respond

Apologies for the delayed response. @iiio2 @kanakdesai

**Comment by noahnsimbe** [comment]
> Port 3000 is used for both the backend as well as the font end? I might be wrong, but using same port for backend as well as the frontend might cause problems.

Yes @kanakdesai I might cause problems. I shall review and make changes.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# syntax=docker/dockerfile:1
+
+FROM node:18-alpine  as base
```
Comment (iiio2) [inline Dockerfile:3]: remove extra space between `alpine` and `as`

File: compose.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+version: '3'
```
Comment (iiio2) [inline compose.yml:1]: use double quote

File: run-compose.sh
Comment (iiio2) [inline run-compose.sh:1]: u may use without shell

**Comment by iiio2** [comment]
@chinweokwu pls test it.

**Pull request closed without merging**